src/commands/contract/migrate.ts

```diff
@@ -27,37 +31,59 @@ export default class ContractMigrate extends Command {
   async run() {
     const { args, flags } = this.parse(ContractMigrate);
 
-    const connections = loadConnections(flags['config-path']);
-    const config = loadConfig(flags['config-path']);
-    const conf = config(flags.network, args.contract);
-
-    const lcd = new LCDClient(connections(flags.network));
-    const signer = await getSigner({
-      network: flags.network,
-      signerId: flags.signer,
-      keysPath: flags['keys-path'],
-      lcd,
-    });
-
-    const codeId = await storeCode({
-      conf,
-      noRebuild: flags['no-rebuild'],
-      contract: args.contract,
-      signer,
-      network: flags.network,
-      refsPath: flags['refs-path'],
-      lcd,
-    });
-
-    migrate({
-      conf,
-      signer,
-      contract: args.contract,
-      codeId,
-      network: flags.network,
-      instanceId: flags['instance-id'],
-      refsPath: flags['refs-path'],
-      lcd,
-    });
+    // Command execution path.
+    const execPath = flags['config-path'];
+
+    // Command to be performed.
+    const command = async () => {
+      const connections = loadConnections(flags['config-path']);
+      const config = loadConfig(flags['config-path']);
+      const conf = config(flags.network, args.contract);
+
+      const lcd = new LCDClient(connections(flags.network));
+      const signer = await getSigner({
+        network: flags.network,
+        signerId: flags.signer,
+        keysPath: flags['keys-path'],
+        lcd,
+      });
+
+      const codeId = await storeCode({
+        conf,
+        noRebuild: flags['no-rebuild'],
+        contract: args.contract,
+        signer,
+        network: flags.network,
+        refsPath: flags['refs-path'],
+        lcd,
+      });
+
+      migrate({
+        conf,
+        signer,
+        contract: args.contract,
+        codeId,
+        network: flags.network,
+        instanceId: flags['instance-id'],
+        refsPath: flags['refs-path'],
+        lcd,
+      });
+    };
+
+    // Error check to be performed upon each backtrack iteration.
+    const errorCheck = () => {
+      if (existsSync('contracts') && !existsSync(join('contracts', args.contract))) {
+        TerrainCLI.error(
+          `Contract '${args.contract}' not available in 'contracts/' directory.`,
+        );
+      }
+    };
```

Comment:
same as this comment: https://github.com/terra-money/terrain/pull/132#discussion_r986039798